alpha1: 10
alpha2: 10
alpha3: 10
alpha4: 10
alpha5: 10
alpha6: 10
